### DISCUSSION TITLE
How can I check the entered password?

### DISCUSSION THREAD

**Discussion by mike2003** [keepassxreboot/keepassxc#7525] [Q&A]
Everywhere when entering a password, I see two fields. They both need to match for me to enter the code correctly. Without this, I cannot enter complex passwords, as there is a high probability of a mistake. I don't see it in this program. Isn't there such an important function here? 
In keepass - **Repeat** field

**Comment by droidmonkey** [comment]
If you must manually enter a password when editing an entry just click the eye or press ctrl+h to reveal the password to check it. We do recommend using the password generator.

**Comment by mike2003** [comment]
No, I don't need it. I want the program to check the complex password itself, as other programs do.
